util/libcrypto.num

```diff
@@ -4379,3 +4379,7 @@ BN_priv_rand_range                      4322	1_1_1	EXIST::FUNCTION:
 ASN1_TIME_normalize                     4323	1_1_1	EXIST::FUNCTION:
 ASN1_TIME_cmp_time_t                    4324	1_1_1	EXIST::FUNCTION:
 ASN1_TIME_compare                       4325	1_1_1	EXIST::FUNCTION:
+ossl_ctype_check                        4326	1_1_1	EXIST::FUNCTION:
+ossl_tolower                            4327	1_1_1	EXIST::FUNCTION:
+ossl_toupper                            4328	1_1_1	EXIST::FUNCTION:
+ossl_isalnum                            4329	1_1_1	EXIST::FUNCTION:
```

Comment:
I'd like to cast a vote *against* exposing this from OpenSSL. For starters libcrypto should be about crypto, not standard functionality replacement. Yes, we do need to treat some cases specially, but that should be internal affairs. As for using this in apps/*.c. I'd say that they, apps/*.c, should serve as an example of how to use OpenSSL by *not* using any of internal/*.h interfaces. Besides it's not given that it's appropriate to mechanically replace *all* occurrences of iswhatever() with their effectively ASCII-centric variants, especially in user-facing apps/*.c.